src/components/Integrations/BigQuery/AuthorizationPage.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect } from 'react';
+import { Loading } from '../../Loading/Loading';
+import { AuthorizationForm } from './ControlPanel/AuthorizationForm';
+import { InjectAppServices } from '../../../services/pure-di';
+import { Promotional } from '../../shared/Promotional/Promotional';
+import { useIntl } from 'react-intl';
+import dataStudioGif from '../../../img/google-data-studio.gif';
+import bigQueryLogo from './styles/bigquery_logo.png';
+
+const AuthorizationLayout = ({ dependencies: { bigQueryClient } }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+  const [data, setData] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [userPlan, setUserPlan] = useState({});
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const result = await bigQueryClient.getEmailsData();
+      if (result.success) {
+        setData(result.value.emails);
+      }
+      const usrPlan = bigQueryClient.getUserPlan();
+      setUserPlan(usrPlan);
+      setLoading(false);
+    };
+
+    fetchData();
+  }, [bigQueryClient]);
+
+  const onSubmit = async (values) => {
+    const emailsData = { emails: [...values.emails] };
+    const { success } = await bigQueryClient.saveEmailsData(emailsData);
+    if (success) {
+      alert('success');
+    } else {
+      alert('error');
+    }
+  };
+
+  if (loading) {
+    return <Loading page />;
+  }
+
+  //TODO implement variable for validate if show promotional or control panel
+  if (userPlan.plan.planName === 'Plus Account') {
```

Comment:
What would happen if the planName changes? shoufd be like this `if(userPlan.plan.planName === 'Plus Account') `@cbernat

Reply:
Maybe you can to do something like this: 
```

if(is not a plus plan) {
    return 'promotional component'
}

return "authorization form"

```
it is easier to read